nushell_completions.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2013-2022 The Cobra Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cobra
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+)
+
+func (c *Command) GenNushellCompletion(w io.Writer, includeDesc bool) error {
```

Comment:
The `includeDesc` variable is not used.

What you need to do is simple, if `includeDesc == false` you want to use `__completeNoDesc` in the script, instead of `__complete`.
If we have a single `cobra_completer` this won't really work as it will affect every program.
Let's see where we end up with having different completers and revisit at that time.

Reply:
We could allow this to be set via an environment variable? Maybe something like this:

```nushell
$env.COBRA_NO_DESC_COMMANDS = [kubectl minikube]
```

Reply:
@marckhouzam, thoughts?

Reply:
Using an environnement variable means it is the end user that will decide if there will be completion descriptions or not. This is not a bad thing, but it doesn’t address the fact that the current proposed API (the function above) does not respect its `includeDesc` argument. 

If we have a single completer for all cobra programs instead of one per program, I don’t think we can do any better than that. But in that case, maybe we should remove the extra useless argument from this function?

Reply:
Removed!